pages/blog/index.page.tsx

```diff
@@ -68,7 +68,7 @@ export default function StaticMarkdownPage({ blogPosts }: { blogPosts: any[] })
       <Head>
         <title>JSON Schema Blog</title>
       </Head>
-      <div className='max-w-[1400px] mx-auto flex flex-col items-center mt-10'>
+      <div className='max-w-[1400px] mx-auto flex flex-col items-center mt-10 overflow-hidden'>
```

Comment:
Can you pls. make it overflow hidden in x direction only. using overflow-x-hidden

Reply:
okay @DhairyaMajmudar